code/game/objects/items/weapons/grenades/grenade.dm

```diff
@@ -97,3 +97,16 @@
 	walk(src, null, null)
 	..()
 	return
+
+/obj/item/grenade/fall_impact(turf/from, turf/dest)
+	..()
+	var/found = dest.contents.Find(/mob/living/carbon/human)
+	if(found)
+		var/mob/living/carbon/human/bonk = dest.contents[found]
+		if(bonk.incapacitated(INCAPACITATION_GROUNDED))
+			return
+		bonk.apply_damage(2, BRUTE, BP_HEAD, FALSE , FALSE, src)
+		bonk.visible_message(SPAN_DANGER("[src] falls from above and bonks [bonk.name] on his head!"), SPAN_DANGER("[src] falls on your head and bounces to the side!"), "You hear something round hitting someone's head", 5)
```

Comment:
```suggestion
		bonk.visible_message(SPAN_DANGER("[src] falls from above and bonks [bonk.name] on \his head!"), SPAN_DANGER("[src] falls on your head and bounces to the side!"), "You hear a dull thud.", 5)
```
A round object hitting someone's head does not make a special sound that you can recognize.